HACKING.md

```diff
@@ -158,12 +158,12 @@ To test the `snapd` REST API daemon on a snappy system you need to
 transfer it to the snappy system and then run:
 
     sudo systemctl stop snapd.service snapd.socket
-    sudo /lib/systemd/systemd-activate -E SNAPD_DEBUG=3 -E SNAP_REEXEC=0 -E SNAPD_DEBUG_HTTP=3 -l /run/snapd.socket -l /run/snapd-snap.socket ./snapd
+    sudo /lib/systemd/systemd-activate -E SNAPD_DEBUG=3 -E SNAPD_DEBUG_HTTP=3 -l /run/snapd.socket -l /run/snapd-snap.socket ./snapd
```

Comment:
I see, it's ok,  mostly because of point 1,  I can imagine scenario when one is trying to run an old locally built snapd, but this reminds me what I was discussing with Chipaca, that is we should log the version when we start snapd (it would help debugging in many situations)